eng/common/scripts/Validate-All-Packages.ps1

```diff
@@ -1,50 +1,252 @@
 [CmdletBinding()]
 Param (
   [Parameter(Mandatory=$True)]
-  [array]$ArtifactList,
+  [array] $PackageInfoFiles,
   [Parameter(Mandatory=$True)]
   [string]$ArtifactPath,
   [Parameter(Mandatory=$True)]
   [string]$RepoRoot,
   [Parameter(Mandatory=$True)]
   [string]$APIKey,
-  [string]$ConfigFileDir,
   [string]$BuildDefinition,
   [string]$PipelineUrl,
   [string]$APIViewUri  = "https://apiview.dev/AutoReview/GetReviewStatus",
   [bool] $IsReleaseBuild = $false
 )
 
+# Validate-All-PackagesV2.ps1 folds in the code that was originally in Validate-Package.ps1
```

Comment:
Looks like this comment is outdated as there is no V2 any longer.

Reply:
Comment removed locally, will be updated with the next commit.